nalej/web
# Organization users card component layout

#### What does this PR do?
The purpose of this task is the creation of the organization's users' card component.
- Layout
- Creation of the users' info card component layout

Organization view [design](https://xd.adobe.com/view/584132d0-d361-4a29-55d4-ae0a08c3a8f7-2e70/screen/32ceb881-648d-445f-b9a3-dc9f9c23e8f5/Organization-Users-list-with-pic-/)

#### Where should the reviewer start?
Checking the code hosted in the organization folder

#### What is missing?
This is a layout feature, all methods used in the .ts files are going to be made in future features

#### How should this be manually tested?
1. `npm start``
2. Navigate to the organization and check the "info "card layout

#### Any background context you want to provide? 
Email field in organization interface is not  already implemented in the backend but will be very soon, so it is used in the layout

#### What are the relevant tickets?

- [NP-2575](https://nalej.atlassian.net/browse/NP-2575)

#### Screenshots (if appropriate)
![Captura de pantalla 2020-01-27 a las 16 35 20](https://user-images.githubusercontent.com/33664787/73188355-09a9b880-4123-11ea-8e26-9e1db2ee8779.png)


#### Questions
